### PR TITLE
add astar and shiden config

### PR DESCRIPTION
**Pull request by gluneau**
Tests that are working are checked:

- [x] Polkadot to Astar for DOT
- [x] Kusama to Shiden for KSM

### CODE REVIEW

**Comment by xlc** [comment]
please merge master to include fix for kusama

File: scripts/configs/shiden.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+endpoints:
+  - wss://astar.api.onfinality.io:443/public-ws
```
Comment (xlc) [inline scripts/configs/shiden.yml:2]: this is not shiden?

File: tests/xcm-transfer/kusama-relay.test.ts [hunk 2]
```diff
@@ -52,6 +53,30 @@ const tests = [
       },
     },
   },
+  // // shiden <-> kusama
```
Comment (xlc) [inline tests/xcm-transfer/kusama-relay.test.ts:56]: do you still want this?

Comment (gluneau) [reply tests/xcm-transfer/kusama-relay.test.ts]: no next time

File: scripts/configs/shiden.yml (hunk 1 above)
Comment (gluneau) [reply scripts/configs/shiden.yml]: it should, I've fixed it.

**Review by xlc — APPROVED** [review]

**Pull request merged**